pkg/ccl/changefeedccl/kvfeed/physical_kv_feed.go

```diff
@@ -107,6 +109,15 @@ func (p rangefeedFactory) Run(ctx context.Context, sink kvevent.Writer, cfg rang
 	return g.Wait()
 }
 
+// quantizeTS returns a new timestamp with the walltime rounded down to the
+// nearest multiple of the quantization granularity.
+func quantizeTS(ts hlc.Timestamp, granularity time.Duration) hlc.Timestamp {
+	return hlc.Timestamp{
```

Comment:
nit: can you move the non-zero check into here or add one? for my peace of mind / to prevent abuse

Reply:
Done.